ofrak_components/ofrak_components_test/test_gzip_component.py

```diff
@@ -86,6 +82,6 @@ async def verify(self, root_resource: Resource) -> None:
                 if e.returncode == 2 or e.returncode == -2:
                     data = e.stdout
                 else:
-                    raise RuntimeError(format_called_process_error(e))
+                    raise
```

Comment:
What error does this raise?

Reply:
A `raise` in an `except` block will re-raise the same error that was caught. It is equivalent to `raise e`